exercises/practice/grains/.meta/example.8th

```diff
@@ -0,0 +1,9 @@
+: square \ n -- n
+  1 64 between !if null? ;then
```

Comment:
As Bruce mentioned, need a `dup` first. And then, if not between, need to `drop` before adding `null` to the stack.
```suggestion
  dup 1 64 between !if drop null ;then
```
As it stands, the `n:1- 2 swap n:^` is operating on the _expected value_: you _should_ be seeing the "PANIC: expected stack depth to be 3" message. Is your main branch up-to-date (or your local copy of the 8th test runner)?

Reply:
Alternately, copy the input to the r-stack:
```forth
: square
  dup>r
  1 64 between !if null ;then
  r> n:1- 2 swap n:^
;
```